src/MagnumPlugins/DdsImporter/Test/DdsImporterTest.cpp

```diff
@@ -68,34 +74,41 @@ DdsImporterTest::DdsImporterTest() {
               &DdsImporterTest::dxt3,
               &DdsImporterTest::dxt5,
 
+              &DdsImporterTest::dxt10data,
+              &DdsImporterTest::dxt10tooShort,
+              &DdsImporterTest::dxt10unsupportedFormat,
+
               &DdsImporterTest::useTwice});
+
+    addInstancedTests({&DdsImporterTest::dxt10formats2d}, 46);
+    addInstancedTests({&DdsImporterTest::dxt10formats3d}, 12);
```

Comment:
Could you use an enum constant for the size and then also use the same constant below there in the `files` struct? Talking from experience, it will prevent annoying problems when extending/shortening the list later. Details [in the docs](http://doc.magnum.graphics/corrade/classCorrade_1_1TestSuite_1_1Tester.html#details).